etl_manager/extract_metadata.py

```diff
@@ -0,0 +1,409 @@
+import cx_Oracle
```

Comment:
I'm not sure about making `cx_Oracle` a dependency of this package. 

It might be worth looking how we did this with `dataengineeringutils3` [here](https://github.com/moj-analytical-services/dataengineeringutils3/blob/master/dataengineeringutils3/db.py#L8-L18). We don't make oracle a dependency but instead accept a connection as a parameter. That way also allows you to generalise it for other connections.

Reply:
Ah, that seems reasonable - so basically remove the 'create connection' function from there and set up the connection in the repo and script that call these functions.